site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -38,9 +39,73 @@ spec:
       # required
       maxReplicas: 20
 ```
+{{% /feature %}}
+{{% feature publishVersion="1.17.0" %}}
+```yaml
+apiVersion: "autoscaling.agones.dev/v1"
+kind: FleetAutoscaler
+# FleetAutoscaler Metadata
+# {{< k8s-api href="#objectmeta-v1-meta" >}}
+metadata:
+  name: fleet-autoscaler-example
+spec:
+  # The name of the fleet to attach to and control. Must be an existing Fleet in the same namespace
+  # as this FleetAutoscaler
+  fleetName: fleet-example
+  # The autoscaling policy
+  policy:
+    # type of the policy. for now, only Buffer is available
+    type: Buffer
+    # parameters of the buffer policy
+    buffer:
+      # Size of a buffer of "ready" game server instances
+      # The FleetAutoscaler will scale the fleet up and down trying to maintain this buffer, 
+      # as instances are being allocated or terminated
+      # it can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%)
+      bufferSize: 5
+      # minimum fleet size to be set by this FleetAutoscaler. 
+      # if not specified, the actual minimum fleet size will be bufferSize
+      minReplicas: 10
+      # maximum fleet size that can be set by this FleetAutoscaler
+      # required
+      maxReplicas: 20
+  # The autoscaling sync strategy
+  sync:
+    # type of the sync. for now, only FixedInterval is available
```

Comment:
```suggestion
  # [Stage:Alpha]
  # [FeatureFlag:CustomFasSyncInterval]
    # type of the sync. for now, only FixedInterval is available
```

Just aligning this with our [other examples](https://github.com/googleforgames/agones/blob/main/examples/gameserver.yaml#L81)

Ooh, that made me remember, can we add these changes in here too:
https://github.com/googleforgames/agones/blob/main/examples/fleet.yaml

Sorry about that, just realised.

Reply:
👍 Thank you for point out! Hope there's no other comments I missed.